MdeModulePkg/Library/BootMaintenanceManagerUiLib/BootMaintenanceManagerCustomizedUiSupport.c

```diff
@@ -381,88 +381,90 @@ BmmListThirdPartyDrivers (
 
   HiiHandles = HiiGetHiiHandles (NULL);
   ASSERT (HiiHandles != NULL);
+  // MU_CHANGE - Ensure HiiHandles are valid before using
+  if (HiiHandles != NULL) {

```

Comment:
Yes, the caller of this function is not checking any of the return statuses:

https://github.com/microsoft/mu_basecore/blob/release/202208/MdeModulePkg/Library/BootMaintenanceManagerUiLib/BootMaintenanceManagerCustomizedUi.c#L47

Reply:
I don't see the point of the return status in this function.

Given what this is supposed to do, I think it should return `EFI_NOT_FOUND` if prerequisites are not found to identify the driver list. An alternative would be to remove the return status since it is effectively not used at the moment.

This is not necessarily needed for your change though, a lot of the BDS code has dormant issues like this.